packages/KDropdownMenu/KDropdownItem.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <li
+    :class="{
+      'has-divider': type !== 'link' && hasDivider,
+      'disabled': type === 'default' && disabled,
+      'danger': isDangerous,
+      'k-dropdown-selected-option': selected
+    }"
+    class="k-dropdown-item w-100"
+  >
+    <router-link
+      v-if="type === 'link' && to"
+      :data-testid="label"
+      :class="{ 'disabled': disabled, 'has-divider': hasDivider }"
+      :to="!disabled ? to : $router.currentRoute.path"
+      class="k-dropdown-item-trigger"
+    >
+      <slot>{{ label }}</slot>
+    </router-link>
```

Comment:
Is it possible to refactor so that links are implemented via slot content instead of utilizing `vue-router` inside Kongponents? I this would be a better long-term solution and would break the peer dependency requirement (unsure if we still use it in other places, but removing would be ideal).

Reply:
I kind of like offering the linking and both `KButton` and `KBreadcrumb` use `router-link` - removing `vue-router` basically makes our breadcrumb component useless.

Reply:
Yeah we already have the peer dependency, so may as well